go-controller/pkg/ovn/policy.go

```diff
@@ -257,30 +262,56 @@ func (oc *Controller) createDefaultDenyPGAndACLs(namespace, policy string, nsInf
 	return nil
 }
 
-func (oc *Controller) setACLLoggingForNamespace(ns string, namespacePolicies map[string]*networkPolicy, portGroupIngressDenyName string, portGroupEgressDenyName string, logSeverityConfig ACLLoggingLevels) error {
+func (oc *Controller) updateACLLoggingForPolicy(np *networkPolicy, logLevel string) error {
+	np.Lock()
+	defer np.Unlock()
+
+	if !np.created {
+		return NetworkPolicyNotCreated
+	}
+	var ovsDBOps []ovsdb.Operation
+	acls := oc.buildNetworkPolicyACLs(np, logLevel)
+	ops, err := libovsdbops.UpdateACLsLoggingOps(oc.nbClient, ovsDBOps, acls...)
+	if err != nil {
+		return err
+	}
+	ovsDBOps = append(ovsDBOps, ops...)
+	_, err = libovsdbops.TransactAndCheck(oc.nbClient, ovsDBOps)
+	return err
+}
+
+func (oc *Controller) setACLLoggingForNamespace(ns string, nsInfo *namespaceInfo) error {
 	var ovsDBOps []ovsdb.Operation
 	for _, policyType := range []knet.PolicyType{knet.PolicyTypeIngress, knet.PolicyTypeEgress} {
-		denyACL, _ := buildDenyACLs(ns, "", targetPortGroupName(portGroupIngressDenyName, portGroupEgressDenyName, policyType), logSeverityConfig.Deny, policyType)
+		denyACL, _ := buildDenyACLs(ns, "", targetPortGroupName(nsInfo.portGroupIngressDenyName, nsInfo.portGroupEgressDenyName, policyType), nsInfo.aclLogging.Deny, policyType)
 		ops, err := libovsdbops.UpdateACLsLoggingOps(oc.nbClient, ovsDBOps, denyACL)
```

Comment:
I did not notice this at first review but also no need for separate ops

`ovsDBOps, err := libovsdbops.UpdateACLsLoggingOps(oc.nbClient, ovsDBOps, denyACL)`

`libovsdbops` work like `append`